opengever/workspace/participation/browser/participants_view.py

```
@@ -0,0 +1,55 @@
+from ftw.keywordwidget import _ as KWMF
+from opengever.workspace import _
+from Products.Five.browser import BrowserView
+from Products.Five.browser.pagetemplatefile import ViewPageTemplateFile
+from zope.i18n import translate
+import json
+import os
+
+
+class ParticipantsView(BrowserView):
+
+    template = ViewPageTemplateFile('templates/participants-view.pt')
+
+    # Do not render vue.js template with PageTemplate engine,
+    # There's actually nothing to process.
+    with open(os.path.join(os.path.dirname(__file__),
+                           'templates',
+                           'table.html'), 'r') as html_file:
+        table_template = html_file.read()
+
+    def __call__(self):
+        return self.template()
+
+    # @property
+    # def table_template(self):
+    #    """Enable for development."""
+    #     with open(os.path.join(os.path.dirname(__file__),
```

✂️ 😏

---

@jone :-( but it is useful for development. If I remove another will fail hard at this point. What so you think?

---

I would not know that this is here, I'll just be annoyed that it does not apply the changes and reboot 😜 

Could we change it so that it just works as expected?
Could we just cache the template as long as the mtime of the file does not change?
Or maybe check the debug mode?

---

Ok: I just remove it for now :-)